internal/lib/container_server.go

```diff
@@ -439,6 +439,11 @@ func (c *ContainerServer) LoadContainer(ctx context.Context, id string) (retErr
 		imgRef = ""
 	}
 
+	platformRuntimePath, ok := m.Annotations[crioann.PlatformRuntimePath]
+	if !ok {
+		platformRuntimePath = ""
+	}
+
```

Comment:
nit: this can be acheieved with 
```
platformRuntimePath, _ := m.Annotations[crioann.PlatformRuntimePath]
```

Reply:
ok, I thought of maintaining the same code styling as this one: https://github.com/cri-o/cri-o/blob/main/internal/lib/container_server.go#L437

Reply:
IMO that one is wrong too, though it predates me so I hadn't noticed

Reply:
chatted offline, my suggestion gives linter errors (that are also wrong IMO :upside_down_face: )